src/cmap/conn/mod.rs

```diff
@@ -165,7 +171,8 @@ impl Connection {
     pub(crate) fn info(&self) -> ConnectionInfo {
         ConnectionInfo {
             id: self.id,
-            server_id: self.server_id,
```

Comment:
I'm of two minds here whether truncation or omission is better when the server_id doesn't fit in an i32.  @isabelatkinson do you have an opinion here?

Reply:
I think it would be a breaking/unexpected change for users who are currently relying on this field to be `Some` (even if it's a truncated value) to stop populating it.

Reply:
I agree with Isabel. The i64 overflow issue affects a small amount of users and would bother more users if we just removed support for the original server_id without warning.